WordPress/src/main/java/org/wordpress/android/ui/themes/ThemeBrowserActivity.java

```diff
@@ -180,6 +186,17 @@ public void onSwipeToRefresh() {
         fetchWpComThemesIfSyncTimedOut(true);
     }
 
+    private void initViewModel() {
+        mViewModel = new ViewModelProvider(this, mViewModelFactory).get(ThemesViewModel.class);
+        mViewModel.setSite(mSite);
```

Comment:
Instead of setting the site directly we may use an initialisation function in the ViewModel to pass it over.

Reply:
Thank you for making this change 👍